public/scss/cart.scss

```diff
@@ -0,0 +1,194 @@
+            font-size: 15px;
+            display: contents;
+            color: white;
+        }
+    }
+
+    a{
+        color: white;
+        display: none;
+    }
+}
+
+.cart-item-list{
+    display: flex;
+    max-width: 1140px;
+    margin-top: 10px;
+    background-color: white;
+    margin: 10px;
+    border-radius: 5px;
+    padding: 0;
+}
+
+.cart-item{
+    display: flex;
+    width: 100%;
+.cart-image img{
+    width: 50%;
+    margin-left: 10px;
+}
+}
+
+.description {
+    padding-top: 10px;
+    margin-right: 60px;
+    width: 100%;
+  }
+   
+  .description span {
+    display: block;
+    font-size: 14px;
+    color: #43484D;
+    font-weight: 400;
+    
+  }
+   
+  .description span:first-child {
+    margin-bottom: 5px;
+  }
+  .description span:last-child {
+    font-weight: 300;
+    margin-top: 8px;
+    color: #86939E;
+  }
+  .item-count{
+      display: flex;
+      padding-top: 10px;
+      #cart-counter {
+          text-align: center;
+          max-width: 50px;
+          border: 0;
+         }
+         p{
+             margin-left: 35px;
+         }
+         .total-price {
+            flex: auto;
+            text-align: right;
+          
+         }
+
+          button[class*=btn] {
+            background-color: Transparent;
+            background-repeat:no-repeat;
+            border: none;
+            cursor:pointer;
+            overflow: hidden;   
+          }
+  }
+  .banner{
+      display: flex;
+      justify-content: center;
+      background-color: white;
+      margin: 10px;
+      border-radius: 5px;
+      img{
+          width: 15%;
+          height: 20%;
+      }
+      p{
+          margin-top: 20px;
+          margin-left: 75px;
+          font-weight: 200;
+          font-size: 17px;
+      }
+  }
+
+.checkout-btn{
+    justify-content: flex-end;
+    background-color: white;
+    margin: 50px 10px 10px 10px;
+    border-radius: 5px;
+    .btn-checkout{
+        border-radius: 5px;
+        width: 100%;
+        margin-top: 10px;
+        font-size: 17px;
+    }
+    p{
+        font-weight: 200;
+        font-size: 17px;
+        padding-top: 10px;
+        text-align: center;
+    }
+}
+@media only screen and (max-width:$breakpoint-tablet) {
+    .banner{
+        p{
+            margin-top: 10px;
+        }
+    }
+}
+@media only screen and (max-width: $breakpoint-mobile) {
+
+    .cart-item{
+    .cart-image {
+        width: 45%;
+        }
+    }
+    svg {
+        width: 17px;
+        height: 17px;
+    }
+    .description {
+        padding-top: 0;
+        margin-right: 0;
+        width: 100%;
+        font-size: 14px;
+    }
+    .item-count{
+        p{
+            margin: 0;
+            padding-left: 10px;
+        }
+
+        button.plus-btn{
+            padding: 0;
+        }
+        button.minus-btn{
+            padding: 0;
+        }
+    }
+    .banner{
+        img{
+            width: 30%;
+        }
+        p{
+            margin: 10px 0px 0px 0px;
```

Comment:
You can use shortcut property here: 

margin: 10px 0 0

Reply:
Done